apps/web/pages/signup.tsx

```diff
@@ -210,6 +210,20 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
     where: {
       token,
     },
+    include: {
+      team: {
+        select: {
+          metadata: true,
+          parentId: true,
+          parent: {
+            select: {
+              slug: true,
+            },
+          },
+          slug: true,
+        },
+      },
+    },
```

Comment:
We now rely on the team assigned to the verification token, not the assigned teamId in the user as that could be null at certain point